packages/js-drive/package.json

```diff
@@ -92,7 +92,7 @@
     "lodash.get": "^4.4.2",
     "lodash.set": "^4.3.2",
     "long": "^5.2.0",
-    "lru-cache": "^5.1.1",
+    "lru-cache": "^6.0.0",
```

Comment:
I would revert it back to 5.1.1. It's a major version bump, and the change is not relevant to PR.